YellowNotes.Api/Startup.cs

```diff
@@ -3,9 +3,16 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace YellowNotes.Api
 {
+    public class AllowedHosts
```

Comment:
separate file

Reply:
I did it without that class right now is that ok?